modules/multi-architecture-enabling-64k-pages.adoc

```diff
@@ -0,0 +1,89 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multi-architecture-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-enabling-64k-pages_{context}"]
+
+= Enabling 64k pages on the {op-system-first} kernel 
+
+You can enable 64k memory page in the {op-system-first} kernel on the 64-bit ARM compute machines in your cluster. The 64k page size kernel specification can be used for large GPU or high memory workloads. This is done using the Machine Config Operator (MCO) which uses a machine config pool to update the kernel. To enable 64k page sizes, you must dedicate a machine config pool for ARM64 to enable on the kernel.
+
+[IMPORTANT]
+====
+Using 64k pages is exclusive to 64-bit ARM architecture compute nodes or clusters installed on 64-bit ARM machines. If you configure the 64k pages kernel on a machine config pool using 64-bit x86 machines, the machine config pool and operator will degrade.
+====
+
+.Prerequisites: 
+* You installed the OpenShift CLI `oc`
+* You created a cluster with compute nodes of different architecture on one of the supported platforms
+
+.Procedure: 
+
+. Label the nodes where you want to run the 64k page size kernel:
+[source,terminal]
++
+----
+$ oc label node <node_name> <label>
+----
++
+[source,terminal]
+----
+$ oc label node worker-arm64-01 node-role.kubernetes.io/worker-64k-pages=
+----
+
+. Create a machine config pool that contains the worker role that uses the ARM64 architecture and the `worker-64k-pages` role:
+[source,yaml]
++
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: worker-64k-pages
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - key: machineconfiguration.openshift.io/role
+        operator: In
+        values:
+        - worker
+        - worker-64k-pages
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/worker-64k-pages: ""
+      kubernetes.io/arch: arm64
+----
+
+. You need to also create a machine config on your compute node to enable `64k-pages`
++
+[source,terminal]
+----
+$ oc create -f <filename>.yaml
+----
+
+. Edit the machine config and add the `64k-pages` parameter to the configuration:
++
+[source,terminal]
+----
+$ oc edit <filename>.yaml
+----
++
+.Example MachineConfig
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: "worker" <1>
```

Comment:
To make this MachineConfig consistent with the other resources in this doc (the pool and the node label) we should use machineconfiguration.openshift.io/role:  "worker-64k-pages" instead.

The current example is a machineconfig that will enable the 64k-pages kernel in the "worker" pool and not in the "worker-64-pages" pool.

The comment <1> tells the customer to edit it if necessary, but wouldn't it be better to use an example value that is consistent with the rest of the resources? I'm afraid of customers copy-pasting the resources and finding out that they are applying the kernel to the wrong pool.